.github/workflows/cmd.yml

```diff
@@ -165,11 +171,52 @@ jobs:
               repo: context.repo.repo,
               content: '+1'
             })
+  
+  # Get PR branch name, because the issue_comment event does not contain the PR branch name
+  get-pr-branch:
+    needs: [clean, fellows]
+    runs-on: ubuntu-latest
+    outputs:
+      pr-branch: ${{ steps.get-pr.outputs.pr_branch }}
+      repo: ${{ steps.get-pr.outputs.repo }}
+    steps:
+      - name: Check if the issue is a PR
+        id: check-pr
+        run: |
+          if [ -n "${{ github.event.issue.pull_request.url }}" ]; then
+            echo "This is a pull request comment"
+          else
+            echo "This is not a pull request comment"
+            exit 1
+          fi
+
+      - name: Get PR Branch Name and Repo
+        if: steps.check-pr.outcome == 'success'
+        id: get-pr
+        uses: actions/github-script@v7
+        with:
+          script: |
+            const pr = await github.rest.pulls.get({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              pull_number: context.issue.number,
+            });
+            const prBranch = pr.data.head.ref;
+            const repo = pr.data.head.repo.full_name;
+            return { pr_branch: prBranch, repo: repo };
+          result-encoding: string
+
+      - name: Use PR Branch Name and Repo
+        run: |
+          echo "The PR branch is ${{ steps.get-pr.outputs.pr_branch }}"
+          echo "The repository is ${{ steps.get-pr.outputs.repo }}"
+
 
   cmd:
-    needs: [clean, fellows]
+    needs: [get-pr-branch, fellows]
     env:
       JOB_NAME: 'cmd'
+      PR_BRANCH: ${{ needs.get-pr-branch.outputs.pr-branch }}
```

Comment:
Where is this being used?

Reply:
nowhere, though to use it but then didn't )) will remove, thanks